config/locales/two_factor_authentication/fr.yml

```
@@ -13,8 +13,9 @@ fr:
       cancel_link: Annuler votre demande
       link: supprimer votre compte
       pending: Vous avez actuellement une demande en attente pour supprimer votre
-        compte.  Il faut compter 24 heures à partir du moment où vous avez fait
-        la demande pour terminer le processus.  Veuillez vérifier plus tard.
+        compte.  Il faut compter %{interval} à partir du moment où vous avez
+        fait la demande pour terminer le processus.  Veuillez vérifier plus
```

Not directly relevant for your changes, but since we're here, should we collapse to single space between sentences for consistency with English? ([related Slack discussion](https://gsa-tts.slack.com/archives/CEUQ9FXNJ/p1706111249614589))

```suggestion
        compte. Il faut compter %{interval} à partir du moment où vous avez
        fait la demande pour terminer le processus. Veuillez vérifier plus
```